src/main/resources/public/src/utils/plugins/i18n.js

```diff
@@ -0,0 +1,97 @@
+            processed_model: 'Processed model | Processed models',
+            concerned_organization: 'Concerned organization | Concerned organizations',
+            processing_since: 'Processing since | Processings since',
+            search: 'Search',
+            by_model: 'By Model',
+            being_processed: 'being processed',
+            creation: 'Creation | Creations',
+            deletion: 'Deletion | Deletions',
+            update: 'Update | Updates',
+            by_organization: 'By Organization',
+            models_processed_for_selected_organization: 'Models processed for selected organization',
+            all: 'All',
+            for_organization: 'for Organization',
+            created_ago: 'Created ago'
+        },
+        fr: {
+            dashboard: 'Tableau de bord',
+            data_request: 'Demande de données',
+            connectors_management: 'Gestion des connecteurs',
+            statistics: 'Statistiques',
+            dc_model: 'DC Modèle',
+            date: 'Date',
+            organization: 'Organisation | Organisations',
+            state: 'Etat',
+            request_access_to_dataset: 'Demande d\'accès à un jeu de données',
+            search_organization: 'Rechercher une organisation',
+            email: 'Email',
+            model: 'Aucun modèle | Modèle | Modèles',
+            search_model: 'Rechercher un modèle',
+            save: 'Enregistrer',
+            search_by_organization: 'Recherche par organisation',
+            application_name: 'Nom d\'application',
+            search_by_application: 'Recherche par application',
+            organization_name: 'Nom d\'organisation',
+            siret_number: 'Numéro SIRET',
+            data_access_check: 'Contrôle d\'accès aux données',
+            authorize_access_to_all_checked_data: 'Autoriser l\'accès à toutes les données vérifiées du modèle "{modelName}"',
+            field: 'Champ | Champs',
+            validate: 'Valider',
+            refuse: 'Refuser',
+            processing_statistics: 'Statistiques de traitement',
+            since: 'Depuis le',
+            processing_completed: 'Traitement terminé | Traitements terminés',
+            processed_model: 'Modèle traité | Modèles traités',
+            concerned_organization: 'Organisation concernée | Organisation concernées',
+            processing_since: 'Traitement depuis le | Traitements depuis le',
+            search: 'Recherche',
+            by_model: 'Par modèle',
+            being_processed: 'en cours de traitement',
+            creation: 'Création | Créations',
+            deletion: 'Effacement | Effacements',
+            update: 'Mise à jour | Mises à jour',
+            by_organization: 'Par organisation',
+            models_processed_for_selected_organization: 'Modèles traités pour l\'organisation sélectionnée',
+            all: 'Tout',
+            for_organization: 'pour l\'organisation',
+            created_ago: 'Créé il y a'
```

Comment:
I'm not fond of that. Maybe we shouldn't add a title to this column. "X jours" or "X minutes" is explicit enough.
Or I can propose "Depuis"